test/new_relic/agent/custom_event_aggregator_test.rb

```diff
@@ -97,6 +97,26 @@ def test_truncates_attributes
       assert_equal(expected, actual)
     end
 
+    def test_does_not_truncates_llm_embedding_input_attribute
```

Comment:
For the two new tests, I think `truncate` should be singular. I think "does not truncate" reads better than "does not truncates".

Reply:
100% agree. I'll update this